django/db/models/expressions.py

```diff
@@ -1541,6 +1541,7 @@ class Subquery(BaseExpression, Combinable):
     template = "(%(subquery)s)"
     contains_aggregate = False
     empty_result_set_value = None
+    subquery = True
```

Comment:
Monkey typing to avoid doing `issubclass(self.annotations[ref], Subquery)` in `get_aggregation` to compute `refs_subquery`.